src/Rings/mpoly-localizations.jl

```diff
@@ -2346,10 +2346,10 @@ quintuple ``(A, I, d, ϕ, θ)`` consisting of
 function as_affine_algebra(
     L::MPolyLocRing{BRT, BRET, RT, RET, 
 			     MPolyPowersOfElement{BRT, BRET, RT, RET}}; 
-    inverse_name::String="θ"
+    inverse_name::VarName=:_0
   ) where {BRT, BRET, RT, RET}
   R = base_ring(L)
-  A, phi, t = _add_variables_first(R, [inverse_name])
+  A, phi, t = _add_variables_first(R, [Symbol(inverse_name)])
```

Comment:
Does that function accept `Vector{Symbol}` instead of `Vector{String}`?

Reply:
IIRC this PR changes it to do so

Reply:
Ah, I found it. Thanks